### ISSUE TITLE
Netlify build image seems outdated

### ISSUE THREAD

**Issue by k-dominik** [simd-everywhere/simde#1005]
Seems like [netlify removed support for Xenial build images](https://answers.netlify.com/t/please-read-end-of-support-for-xenial-build-image-everything-you-need-to-know/68239)

example build log: https://app.netlify.com/sites/simde/deploys/6442aeb00c95250007d63111

**Comment by mr-c** [comment]
Yep, I can't fix it and I haven't heard from @nemequ lately ; sorry
